examples/widgets/actionbar.py

```diff
@@ -30,4 +30,3 @@
             ActionButton:
                 text: 'Btn7'
 '''))
-
```

Comment:
"./examples/widgets/actionbar.py:33:1: W391 blank line at end of file
"
owing to this error i did so. :)